### PR TITLE
Coveralls

### PR DESCRIPTION
**Pull request by michaelachrisco**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,5 +1,6 @@
 source 'https://rubygems.org'
 
-gem 'codeclimate-test-reporter', :group => :test, :require => nil
+gem 'codeclimate-test-reporter', :group => :test, :require => false
```
Comment (josephbridgwaterrowe) [inline Gemfile]: You can remove this.

File: Gemfile [hunk 2]
```diff
@@ -1,5 +1,6 @@
...
+gem 'coveralls', :require => false
```
Comment (josephbridgwaterrowe) [inline Gemfile]: And move this into the test group?

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -1,5 +1,8 @@
 require 'codeclimate-test-reporter'
```
Comment (josephbridgwaterrowe) [inline spec/spec_helper.rb]: And we won't need the code climate lines here.

File: README.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-[![Code Climate](https://codeclimate.com/github/michaelachrisco/gman_client/badges/gpa.svg)](https://codeclimate.com/github/michaelachrisco/gman_client) [![Test Coverage](https://codeclimate.com/github/michaelachrisco/gman_client/badges/coverage.svg)](https://codeclimate.com/github/michaelachrisco/gman_client)
+[![Build Status](https://travis-ci.org/westernmilling/gman_client.svg?branch=master)](https://travis-ci.org/westernmilling/gman_client) [![Code Climate](https://codeclimate.com/github/michaelachrisco/gman_client/badges/gpa.svg)](https://codeclimate.com/github/michaelachrisco/gman_client) [![Coverage Status](https://coveralls.io/repos/westernmilling/gman_client/badge.svg)](https://coveralls.io/r/westernmilling/gman_client)
```
Comment (josephbridgwaterrowe) [inline README.md]: No code climate coverage please. :smile:

File: Gemfile (hunk 1 above)
Comment (michaelachrisco) [inline Gemfile]: Any reason we are not going to use code climate?

**Pull request merged**